src/containerapp/azext_containerapp/_validators.py

```diff
@@ -215,3 +220,49 @@ def validate_timeout_in_seconds(cmd, namespace):
     if timeout_in_seconds is not None:
         if timeout_in_seconds < 0 or timeout_in_seconds > 60:
             raise ValidationError("timeout in seconds must be in range [0, 60].")
+
+
+def validate_debug(cmd, namespace):
+    print("Validating...")

```

Comment:
Is it good practice to use raw print, rather than the logging capability of CLI?